core/src/test/java/org/opensearch/sql/expression/datetime/DateTimeFunctionTest.java

```diff
@@ -1355,6 +1359,103 @@ void testDateFormat(DateFormatTester dft) {
     assertEquals(dft.getFormatted(), eval(expr).stringValue());
   }
 
+  private static Stream<Arguments> getTestDataForTimeFormat() {
```

Comment:
Please, add tests for both `DATE_FORMAT` and `TIME_FORMAT` where
* first argument is `DATE` and `TIME`
* second argument has string and numeric matchers for date and for time

Consider moving tests to a separate class

Reply:
Fixed in https://github.com/Bit-Quill/opensearch-project-sql/pull/204/commits/8a2279a9734e063e7105fb7915f0fa0fb732a936